Clicker/Views/Cards/AnswerCard.swift

```diff
@@ -18,7 +18,7 @@ class AnswerCard: UICollectionViewCell, CardDelegate, SocketDelegate {
     var choice: Int?
     var poll: Poll!
     var socket: Socket!
-    var cardType: CardType!
+    var cardType: PollState!
```

Comment:
do we need this if we have the `poll` variable which has `PollState`?

Reply:
You're right, that would be a smarter implementation. We previously had a `CardType` enum with the same exact values as the `PollState` enum so i removed the `CardType` enum and replaced it with `PollState` where it was previously being used, but didn't realize we are already storing this information in the `poll` object.

Reply:
Oh yeah I remember we had that when we were doing stuff for BOOM haha